README.rst

```diff
@@ -17,37 +17,19 @@ We need some system packages to be available for this package to be fully functi
 - poppler_utils (pdf)
 - html-tidy (cleans up the html that will be converted to pdf)
 
+Solr: cave canem
```

Comment:
I am pretty sure you can configure the port and avoid the problem.

Reply:
Oh I tried. But the actual startup ignores any custom port and starts with the buildout configured port.
https://github.com/collective/collective.solr/blob/1d65570fd9270d465317469796f17a7b351bd4d5/src/collective/solr/testing.py#L61

In order to keep *this* PR small and focused I've extracted fixing that into a subtask https://github.com/syslabcom/scrum/issues/3974